### DISCUSSION TITLE
Position and responsive size Popup

### DISCUSSION THREAD

**Discussion by DiamondKesha** [CommunityToolkit/Maui#551] [Q&A] [answered]
Hello. How i can make popup like this (first image)? I'm trying to add HorizontalOptions="Fill" VerticalOptions="end", but when i add "Fill" my popup going to center and didn't fill horizontal. And how can I make a Popup have an automatic (responsive) size?
![image](https://user-images.githubusercontent.com/45795685/185086169-f778ef61-7551-48ff-b50c-bf5c544ddfb1.png)

**Comment by pictos** [comment, marked as answer]
Hey, @DiamondKesha today our popup doesn't allow that. This looks like a BottomSheet to me.

**Reply by DiamondKesha** [reply]
Okay, thank you.

**Comment by CliffAgius** [comment]
Closed as answered
